install_config/cluster_metrics.adoc

```diff
@@ -319,6 +331,24 @@ properly.
 
 |===
 
+The Heapster component uses the service name DNS registry to connect to Hawkular Metrics.
+In the metrics code, update the URL used by Heapster to connect to Hawkular Metrics. For example, change:
+
+----
+--sink=hawkular:https://hawkular-metrics:443?tenant=_system&;....
+----
+
+to:
+
+----
+ --sink=hawkular:https://${CUSTOM_HOSTNAME}:443?tenant=_system&;....
+----
+
+`--sink` is hard-coded in the deployer image as the *_heapster.yaml_* template
+is called. The
+link:https://github.com/kubernetes/heapster/blob/master/docs/sink-configuration.md#L5[sink]
+option is passed to the Heapster service that in running.
+
```

Comment:
I would leave out everything around --sink option.  And just say that the "hawkular-metrics" URL is hard coded. 

This name will get the search domain attached and resolve to the service IP